translations/ui-tenant-settings/en.json

```diff
@@ -121,6 +121,24 @@
   "settings.intervalPeriod.weeks": "Weeks",
   "settings.intervalPeriod.months": "Months",
 
+
+  "settings.pluginNames.create-inventory-records": "Create inventory records",
+  "settings.pluginNames.find-user": "Find User",
```

Comment:
FOLIO uses sentence case for everything, so for `find-user` use "Find user". For `find-eresource` use "Find e-resource" (singular). But acronyms (like FOLIO and PO) should be all-caps, so for `find-po-line` use "Find PO line", as you did for `find-erm...`. 

For the `finc-` plugins, what do you think about expanding the labels to "finance"?

Reply:
I honestly wasn't sure what "finc" stood for-I'll expand them to "finance."